SPECS/glusterfs/glusterfs.spec

```diff
@@ -74,74 +81,116 @@
 ##-----------------------------------------------------------------------------
 ## All %%global definitions should be placed here and keep them sorted
 ##
+# selinux booleans whose defalut value needs modification
+# these booleans will be consumed by "%%selinux_set_booleans" macro.
+%global _with_systemd true
```

Comment:
What is this part about? I couldn't find `_with_systemd` inside the macros file defining `selinux_set_booleans`. Can you explain how this is used in this spec?

Reply:
i think this is a misplaced comment.  systemd has nothing to do with selinux.